interface/modules/custom_modules/oe-module-claimrev-connect/public/debug-info.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
needs header like lines 3-11 of interface/modules/custom_modules/oe-module-claimrev-connect/openemr.bootstrap.php

Reply:
done